pkg/controller/webhook/kubefedconfig/webhook.go

```diff
@@ -17,20 +17,17 @@ limitations under the License.
 package kubefedconfig
 
 import (
+	"context"
 	"encoding/json"
 	"fmt"
 	"net/http"
 	"reflect"
-	"strings"
-	"sync"
 
-	"github.com/openshift/generic-admission-server/pkg/apiserver"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
```

Comment:
No need for this empty line and please sort out ordering of imports in block below when this is added.

```suggestion
```